diagram.md

```diff
@@ -2,240 +2,284 @@
+
+```
++--------------------------------+
+| SeekHead | Seek | SeekID       |
+|          |      |--------------|
+|          |      | SeekPosition |
++--------------------------------+
+```
+Figure: Representation of a `SeekHead Element`.
+
+The `Info Element` contains vital information for identifying the whole `Segment`. This includes the title for the `Segment`, a randomly generated unique identifier so that the file can be identified around the world, and if it is part of a series of `Segments`, the unique identifier(s) of any linked `Segments`.
+
+```
++-------------------------+
+| Info | SegmentUID       |
+|      |------------------|
+|      | SegmentFilename  |
+|      |------------------|
+|      | PrevUID          |
+|      |------------------|
+|      | PrevFilename     |
+|      |------------------|
+|      | NextUID          |
+|      |------------------|
+|      | NextFilename     |
+|      |------------------|
+|      | SegmentFamily    |
+|      |------------------|
+|      | ChapterTranslate |
+|      |------------------|
+|      | TimecodeScale    |
+|      |------------------|
+|      | Duration         |
+|      |------------------|
+|      | DateUTC          |
+|      |------------------|
+|      | Title            |
+|      |------------------|
+|      | MuxingApp        |
+|      |------------------|
+|      | WritingApp       |
+|-------------------------|
+```
+Figure: Representation of a `Info Element` and its `Child Elements`.
+
+The `Tracks Elements` tells us the technical details of what is in each track. For instance, is it a video, audio or subtitle track? What resolution is the video? What sample rate is the audio? The `Tracks Elements` can store the name, number, unique identifier, language, and type (audio, video, subtitles, etc) of each track. The `Tracks Element` also identifies what codec to use to decode the track and has the codec's private data for the track.
+
+```
++------------------------------------+
+| Tracks | TrackEntry | TrackNumber  |
+|        |            |--------------|
+|        |            | TrackUID     |
+|        |            |--------------|
+|        |            | TrackType    |
+|        |            |--------------|
+|        |            | Name         |
+|        |            |--------------|
+|        |            | Language     |
+|        |            |--------------|
+|        |            | CodecID      |
+|        |            |--------------|
+|        |            | CodecPrivate |
+|        |            |--------------|
+|        |            | CodecName    |
+|        |            |--------------|-------------------+
```

Comment:
In the very first table you've made the `EBML Head` entry span the width of the whole table, but starting with this table you don't make the upper elements as wide as the table. I think this should be consistent over all tables.